pyvenv.el

```diff
@@ -145,6 +154,34 @@ This is usually the base name of `pyvenv-virtual-env'.")
 (defvar pyvenv-old-eshell-path nil
   "The old eshell path before the last activate.")
 
+
+(defun pyvenv-create ()
+  "Create virtualenv."
+  (interactive)
+  (let ((create-venv-command)
+        (python-executable)
+        (venv-name)
+        (venv-dir))
+    (run-hooks 'pyvenv-pre-create-hooks)
+    (setq venv-name (read-string "Enter virtualenv name: "))
+    (setq venv-dir (concat (file-name-as-directory (pyvenv-workon-home))
+                           venv-name))
+    (unless (file-exists-p venv-dir)
+      (setq python-executable
+            (read-file-name "Enter path of python interpreter: "
+                            (file-name-directory (executable-find "python"))
+                            "python"
+                            nil
+                            "python"))
```

Comment:
It would be nice to use `interactive` to get all of these arguments, to be able to use this function non-interactively, too.

Reply:
It's able to use as non-interactive function, I've apply at this manner because I want use as non-interactive without passing arguments at the function call, so I don't have idea how can implement argument asking on interactive without passing parameters at function call

Reply:
so can you give me an example?

Reply:
http://www.gnu.org/software/emacs/manual/html_node/elisp/Interactive-Examples.html has a few. You can use `(interactive (list (read-from-minibuffer "Arg 1: ") (read-from-minibuffer "Arg 2: "))` to tell Emacs that your command needs two arguments, and those should be read from the minibuffer, say.